tests/support/CliHelper.php

```diff
@@ -20,7 +20,7 @@ public function _after(\Codeception\TestInterface $test)
     {
         codecept_debug('deleting dirs');
         $this->getModule('Filesystem')->deleteDir(codecept_data_dir() . 'sandbox');
-        chdir(\Codeception\Configuration::projectDir());
+        $this->getModule('Filesystem')->amInPath(codecept_root_dir());
```

Comment:
This change broke some unit tests that were relying on matching relative tests when `unit` suite is executed after `cli`

Reply:
Actually the problem was that `amInPath` called in `ConfigParamsCest::_beforeClass` changed working directory for all tests.